src/bioversions/sources/ncit.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""A getter for the NCI Thesaurus."""
+
+from ..utils import Getter, VersionType, get_soup
+
+__all__ = [
+    "NCItGetter",
+]
+
+URL = "https://ncithesaurus.nci.nih.gov/ncitbrowser/"
+
+
+class NCItGetter(Getter):
+    """A getter for the NCI Thesaurus."""
+
+    bioregistry_id = "ncit"
+    name = "National Cancer Institute Thesaurus"
+    version_type = VersionType.other
+
+    def get(self) -> str:
+        """Get the latest NCIt version number."""
+        soup = get_soup(URL)
+        version = soup.find("input", {"id": "version"}).attrs["value"]
+        return version
```

Comment:
it appears there's also a release date on this page. The text reads `Version:21.08e (Release date:2021-08-30)`. If you parse the date too, you could return them both in a dictionary like in the [DisGeNet getter](https://github.com/biopragmatics/bioversions/blob/main/src/bioversions/sources/disgenet.py)

Reply:
Done!